Header header
PoseWithAssoc[] relative_poses
RangeWithAssoc[] range_measurements
PoseWithAssoc[] pose_priors
